oss-internship-2020/libtiff/example/main_sandboxed.cc

```diff
@@ -0,0 +1,313 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <array>
+#include <cstdint>
+#include <cstdlib>
+#include <cstring>
+
+#include "../sandboxed.h"
```

Comment:
To make things easier for us, can you please add `// NOLINT(build/include)` here and to all includes for files that _you_ added?